### ISSUE TITLE
npm@3.10 npm-scripts regression

### ISSUE THREAD

**Issue by necolas** [npm/npm#13429]
#### I'm opening this issue because:

  - [x] npm is doing something I don't understand.

#### What's going wrong?

npm@3.10 is going into an infinite loop when running an npm-script with the same name as a build-in command.

#### How can the CLI team reproduce the problem?

Create a script called `shrinkwrap` than runs `npm shrinkwrap`. Cmd: `npm run shrinkwrap`. npm@3.9 will run the script fine. npm@3.10 will go into an infinite loop.

### supporting information:

 - `npm -v` prints: 3.10.5
 - `node -v` prints: 6.3.0

**Comment by KenanY** [comment]
Result of `git bisect`:

```
e8c80f20bfd5d1618e85dbab41660d6f3e5ce405 is the first bad commit
commit e8c80f20bfd5d1618e85dbab41660d6f3e5ce405
Author: Simen Bekkhus <snip>
Date:   Tue May 24 11:20:21 2016 +0200

    shrinkwrap: Add new "shrinkwrap" lifecycle script
    
    Fixes: #10744
    Credit: @SimenB
    PR-URL: https://github.com/npm/npm/pull/12814

:040000 040000 4634c3c0b10097c33467e19e4a20ff079b4d9c5f d6608523476dd84cabb4e7cef60261b166cef6cb M	doc
:040000 040000 729d90dcbbea3a54337d02969f05dbdac470a286 4f226d83440b5b117ba15ef728887784c21b8e16 M	lib
:040000 040000 525bbb6c40f15851b86a3f78c8712bc17a888ca6 6e5208f8eb226e8440bfb313615b2c89de0470ae M	test
```

Actually looks like this is a duplicate of https://github.com/npm/npm/issues/13253.

**Comment by othiym23** [comment]
> Actually looks like this is a duplicate of #13253.

I believe it is! Thanks for the sleuthing, @KenanY. Closing as resolved.
